### ISSUE TITLE
Add option to delete Astropy cache

### ISSUE THREAD

**Issue by troyraen** [fornax-navo/fornax-demo-notebooks#220]
On the SMCE, users have 10G of disk space by default and I've hit the limit a couple of times. Recently I went hunting deeper for what's taking space and found almost 5G in the Astropy cache:

```
(science_demo) jovyan@jupyter-raen:~$ du -sh ${HOME}/.astropy/*
4.6G    /home/jovyan/.astropy/cache
4.0K    /home/jovyan/.astropy/config
```

I wonder if we should add an option to the end of each notebook to delete this cache using https://docs.astropy.org/en/stable/api/astropy.utils.data.clear_download_cache.html.

**Comment by bsipocz** [comment]
This cache might be useful when working with the very same datasets, but also, I would not think it's a bad idea to turn caching off in the queries themselves.

**Comment by troyraen** [comment]
I would prefer the option of deleting the cache over the option to turn off caching for a particular query. Reason is because I (or the user) may want to use the cache for a few runs, but not be stuck with it forever (or until they go hunting and find/delete it themselves). As it is now, the user doesn't get any indication that such a large amount of data is being cached, plus it's in a hidden directory where they probably won't go looking if/when they run out of space and need to delete things.

**Comment by bsipocz** [comment]
OK, that sounds good. We're also adding more caching docs to astroquery, hopefully that helps, too.

**Comment by troyraen** [comment]
I deleted my cache and executed all the notebooks in this repo that will currently run. They only added about 1M total to `~/.astropy/cache`.

- 800K : forced_photometry
- 30K : light_curve_generator
- 160K : scale_up
- 0 : light_curve_classifier

Not sure how I ended up with 5G in there since I basically only use SMCE to run these notebooks. Doing `get_sdss_sample` (light_curve) with 50,000 objects adds another 2.5M to the cache, but that's still a long way from 5G.

Those numbers are small enough that I don't think we need to put extra work into the notebooks for it.
